<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMyDialog</class>
 <widget class="QDialog" name="CMyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="doneBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>done</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>reject</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>80</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>accept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
